mypy/build.py

```diff
@@ -674,6 +674,7 @@ def __init__(
             if options.custom_typeshed_dir is not None:
                 # Check if module lives under custom_typeshed_dir subtree
                 custom_typeshed_dir = os.path.abspath(options.custom_typeshed_dir)
+                path = os.path.abspath(path)
                 if os.path.commonpath((path, custom_typeshed_dir)) == custom_typeshed_dir:
```

Comment:
I believe it still can through `ValueError` on Windows if path and `custom_typeshed_dir` live on different drives.

Reply:
Good point (though I'm tempted to say that anybody trying to do that deserves what they're getting)